### ISSUE TITLE
Training loss jumping up when resuming training.

### ISSUE THREAD

**Issue by leon-herbrik** [valeoai/Maskgit-pytorch#15]
Hey there,

when I load the model and optimizer state dict from a checkpoint and try to resume training, the training loss suddenly spikes up, removing a lot of the progress the previous training run brought. After a while it goes down again, but the training process is set back by a large margin.

Would you, by chance, know what causes this behavior?

Thanks a lot in advance!

**Comment by llvictorll** [comment]
Hello, 

I haven't noticed this in my previous experiments, but maybe lowering the learning rate or doing learning rate warm-up for the first 1000~5000 steps can help stabilize the fine tuning? 

Best, 

Victor

**Comment by leon-herbrik** [comment]
I'll try that, thanks for the idea!
I am currently fine-tuning the ImageNet base on the Ego4D dataset and the qualitative results are already looking quite good.

**Comment by llvictorll** [comment]
Good luck with that! Also the domain gap could be a reason for the loss jumping for the first steps, as ImageNet and EgoD4 are quite different.
